doc/source/whatsnew/v0.20.0.txt

```diff
@@ -26,6 +26,8 @@ Check the :ref:`API Changes <whatsnew_0200.api_breaking>` and :ref:`deprecations
 New features
 ~~~~~~~~~~~~
 
+- Added subtraction from FrozenLists (:issue:`15475`)
```

Comment:
this is called difference 

i think might be nice to add an example to th docs as well